src/app/components/AdvancedSearch/AdvancedSearch.jsx

```diff
@@ -0,0 +1,461 @@
+                        value={this.state.filters.years.end}
+                        min={this.state.filters.years.start}
+                      />
+                    </div>
+                  </fieldset>
+                </div>
+
+                <div className="tablet:grid-col-6">
+                  <fieldset className="usa-fieldset grid-container width-full margin-x-0 padding-x-0 margin-bottom-2">
+                    <legend className="usa-legend font-sans-lg sub-legend">Format</legend>
+                    <div className="grid-row usa-label">
+                      <Checkbox
+                        className="usa-checkbox tablet:grid-col-12"
+                        labelClass="usa-checkbox__label"
+                        inputClass="usa-checkbox__input"
+                        id="epub"
+                        isSelected={this.state.filters.format.epub}
+                        onChange={this.onFormatChange}
+                        label="ePub"
+                        name="epub"
+                      />
+                    </div>
+                    <div className="grid-row">
+                      <Checkbox
+                        className="usa-checkbox tablet:grid-col-12"
+                        labelClass="usa-checkbox__label"
+                        inputClass="usa-checkbox__input"
+                        id="pdf"
+                        isSelected={this.state.filters.format.pdf}
+                        onChange={this.onFormatChange}
+                        label="PDF"
+                        name="pdf"
+                      />
+                    </div>
+                    <div className="grid-row">
+                      <Checkbox
+                        className="usa-checkbox tablet:grid-col-12"
+                        labelClass="usa-checkbox__label"
+                        inputClass="usa-checkbox__input"
+                        id="html"
+                        isSelected={this.state.filters.format.html}
+                        onChange={this.onFormatChange}
+                        label="Html"
+                        name="html"
+                      />
+                    </div>
+                  </fieldset>
+                </div>
+              </div>
+
+              <div className="grid-row grid-gap">
+                <div className="tablet:grid-col-6">
+                  <div className="grid-row grid-gap">
+                    <div className="tablet:grid-col-6">
+                      <input
+                        className="usa-button width-full usa-label"
+                        type="submit"
+                        value="Search"
+                        readOnly
+                        onClick={this.submitSearchRequest}
+                      />
+                    </div>
+                    <div className="tablet:grid-col-6">
+                      <input
+                        className="usa-button usa-button--outline width-full usa-label"
```

Comment:
Nothing happens when I click this button. I'm not sure if its failing to trigger or if the app is automatically filling back in the information I input when I hit `Clear`.

Reply:
My bad. I totally forgot about that button :)